R/aedseo.R

```diff
@@ -79,6 +79,16 @@ aedseo <- function(
       # TODO: #10 Include negative.binomial regressions. @telkamp7
     ),
     na_fraction_allowed = 0.4) {
+  # Check input arguments
+  coll <- checkmate::makeAssertCollection()
+  checkmate::assert_data_frame(tsd)
+  checkmate::assert_class(tsd, "aedseo_tsd")
+  checkmate::assert_names(names(tsd), identical.to = c("time", "observed"))
```

Comment:
Personally I prefer to use `colnames` instead of `names` for data.frame like objects